### PR TITLE
[BTRX-629] Project Attestation value

### PR DESCRIPTION
**Pull request by tadeork**
## Addresses
Adds attestation to form in New Project and Review Project.
https://broadinstitute.atlassian.net/browse/BTRX-629
## Changes
Create new value to Legacy projects: attestation in true.
## Testing

---

- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Get a thumb from @rushtong
- [ ] **Submitter**: Merge to develop using github's "Squash and Merge" feature
- [ ] **Submitter**: Delete branch after merge

### CODE REVIEW

File: src/main/webapp/project/NewProjectGeneralData.js [hunk 1]
```diff
@@ -194,30 +203,30 @@ export const NewProjectGeneralData = hh(class NewProjectGeneralData extends Comp
...
+              label: "Broad Project Manager",
+              isDisabled: false,
+              loadOptions: this.loadUsersOptions,
+              handleChange: this.handleProjectManagerChange,
+              value: this.state.formData.projectManager,
+              placeholder: "Start typing the Project Manager Name",
+              isMulti: false,
+              edit: false
+            }),
+          ]),
```
Comment (rushtong) [inline src/main/webapp/project/NewProjectGeneralData.js]: This looks like a lot of white space changes with no purpose. I have seen a lot of this in different PRs. I think the team needs to agree on a standard and set up your IDEs so you're not overwriting each others' changes for no reason. The volume of these kinds of changes is really too much and just adds noise to everyone's PRs.

File: src/main/webapp/projectReview/ProjectReview.js [hunk 2]
```diff
@@ -195,12 +200,12 @@ class ProjectReview extends Component {
...
-          {key: 'feeForServiceWork', answer: null},
-          {key: 'areSamplesComingFromEEAA', answer: null},
-          {key: 'isCollaboratorProvidingGoodService', answer: null},
-          {key: 'isConsentUnambiguous', answer: null}
+          { key: 'individualDataSourced', answer: null },
+          { key: 'isLinkMaintained', answer: null },
+          { key: 'feeForServiceWork', answer: null },
+          { key: 'areSamplesComingFromEEAA', answer: null },
+          { key: 'isCollaboratorProvidingGoodService', answer: null },
+          { key: 'isConsentUnambiguous', answer: null }
```
Comment (rushtong) [inline src/main/webapp/projectReview/ProjectReview.js]: Nit: pure whitespace changes.

File: src/main/webapp/project/NewProjectGeneralData.js (hunk 1 above)
Comment (tadeork) [reply src/main/webapp/project/NewProjectGeneralData.js]: We're discussing with the team to implement the same linting template so this stops to happen.

Comment (rushtong) [reply src/main/webapp/project/NewProjectGeneralData.js]: There are lots of examples out there ... 
* Grails plugin https://grails.org/plugin/jslint
* Gradle plugin: https://github.com/nebula-plugins/gradle-lint-plugin
* Intellij plugin: https://www.jetbrains.com/help/idea/jslint.html

File: grails-app/migrations/changesets/changelog-10.0.xml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+    <changeSet id="changelog-10.0-1" author="tadeork">
+        <sql>INSERT INTO issue_extra_property (name, value, project_key, issue_id, version)
+            SELECT 'attestation', 'true', i.project_key, i.id, 0
+            FROM issue i
+            WHERE type != 'Consent Group' and approval_status = 'Legacy';</sql>
+    </changeSet>
+</databaseChangeLog>
```
Comment (rushtong) [inline grails-app/migrations/changesets/changelog-10.0.xml]: 👍 Beautiful!

Comment (tadeork) [reply grails-app/migrations/changesets/changelog-10.0.xml]: great thanks

**Pull request merged**